hack/local-up-cluster.sh

```diff
@@ -638,7 +638,7 @@ function start_apiserver {
 
 function start_controller_manager {
     node_cidr_args=""
-    if [[ "${NET_PLUGIN}" == "kubenet" ]]; then
+    if [[ "${NET_PLUGIN}" == "kubenet" || "${NET_PLUGIN}" == "cni" ]]; then
```

Comment:
Not all CNI plugins will actually want --allocate-node-cidrs, it really depends on what happens to be in /etc/cni/net.d or what happens to get written there when the net plugin's daemonset starts.  So I don't think we can assume CNI==allocate-node-cidrs.  Maybe have another environment variable for local-up-cluster.sh that turns this option on if NET_PLUGIN=cni?